diff --git a/node_modules/@chakra-ui/cli/dist/scripts/read-theme-file.worker.js b/node_modules/@chakra-ui/cli/dist/scripts/read-theme-file.worker.js
index 937cf0d..7dcc0c0 100644
--- a/node_modules/@chakra-ui/cli/dist/scripts/read-theme-file.worker.js
+++ b/node_modules/@chakra-ui/cli/dist/scripts/read-theme-file.worker.js
@@ -50,7 +50,8 @@ async function readTheme(themeFilePath) {
       project: tsConfig.configFileAbsolutePath,
       compilerOptions: {
         module: "CommonJS",
-        esModuleInterop: true
+        esModuleInterop: true,
+        jsx: 'react'
       },
       transpileOnly: true,
       swc: true
